src/server/tiered_storage.cc

```diff
@@ -194,24 +199,25 @@ void TieredStorageV2::Stash(string_view key, PrimeValue* value) {
     visit([this](auto id) { op_manager_->ClearIoPending(id); }, id);
   }
 }
-void TieredStorageV2::Delete(string_view key, PrimeValue* value) {
-  if (value->IsExternal()) {
-    tiering::DiskSegment segment = value->GetExternalSlice();
-    if (segment.length >= kMinOccupancySize) {
-      op_manager_->Delete(segment);
-    } else if (auto bin = bins_->Delete(segment); bin) {
-      op_manager_->Delete(*bin);
-    }
-    value->Reset();
-  } else {
-    DCHECK(value->HasIoPending());
-    if (value->Size() >= kMinOccupancySize) {
-      op_manager_->Delete(key);
-    } else if (auto bin = bins_->Delete(key); bin) {
-      op_manager_->Delete(*bin);
-    }
-    value->SetIoPending(false);
+void TieredStorageV2::Delete(DbIndex dbid, string_view key, PrimeValue* value) {
+  DCHECK(value->IsExternal());
```

Comment:
you do not use dbid, key. is it expected?

Reply:
Actually I don't use the key either 😆